### PR TITLE
nic 10k inventory; WIP

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

File: fab/inventory/icds [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# ToDo update datavol_device vars
```
Comment (sravfeyn) [inline fab/inventory/icds:1]: I will figure the right disks once the storage issue is resolved with NIC

File: fab/inventory/icds [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+# Prefix 'wc21casP-' to node name to get corresponding vm code given on NIC cloud
+
+[pg0]
+10.247.24.12 datavol_device=/dev/sdb
```
Comment (sravfeyn) [inline fab/inventory/icds:5]: What should be `hot_standby_server`?

**Comment by sravfeyn** [comment]
@snopoke

**Comment by benrudolph** [comment]
it's happening! 😲

**Comment by czue** [comment]
@sravfeyn is this PR still relevant? my understanding is latest changes are now on this branch: https://github.com/dimagi/commcare-hq-deploy/compare/ce/icds-inventory

**Comment by sravfeyn** [comment]
Not anymore, Cal branched off. Closing this out

**Pull request closed without merging**